app/_src/kubernetes-ingress-controller/plugins/acl.md

```diff
@@ -106,24 +106,32 @@ Fz/+NmBYpY72Q+XtoszN4E1QUsk1InJ3Wf6hZm3z/CKZLbKIn/UTYTjzKIBPQdLX
 C6V0e/O3LEuJrP+XrEndtLsCAwEAAQ==
 -----END PUBLIC KEY-----{% endcapture %}
 
-1. Create secrets by replacing the RSA key strings with your own from jwt.io. The credentials are stored in Secrets with a `konghq.com/credential` label indicating the type of credential.
+1. Create secrets by replacing the RSA key strings with your own from jwt.io.
+   The credentials are stored in Secrets with a `konghq.com/credential` label indicating the type of credential.
+   Please note that we're adding the labels to the secrets before applying them against the API server to validate upon creation.
 
     ```bash
     kubectl create secret \
       generic admin-jwt \
+      --dry-run="client" \
+      --output yaml \
       --from-literal=key="admin-issuer" \
       --from-literal=algorithm=RS256 \ {% if_version lte:3.3.x %}
       --from-literal=secret="dummy" \ {%- endif_version %}
-      --from-literal=rsa_public_key="{{ public_key }}"
-    kubectl label secret admin-jwt konghq.com/credential=jwt
+      --from-literal=rsa_public_key="{{ public_key }}" | \
+      yq '.metadata += {"labels": {"konghq.com/credential": "jwt"}}' | \
```

Comment:
We don’t assume yq is installed on machines as far as I know. Is this a safe assumption for KIC users?

Reply:
Wasn't aware of this. I believe we should be safe to assume this but let's change this for our users just now.

I've attempted to use `kubectl apply` in c43dbded09f0b9679d2179c34047dda749743e36.

PTAL if that works out for you.

Reply:
Much better. I find `apply` better than `kubectl create` in docs in general